demo.css

```diff
@@ -0,0 +1,52 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: var(--grey-10);
+  font-family: 'Fira Sans', sans-serif;
```

Comment:
We are starting to slowly deprecate Fira Sans in favor of system fonts, we could do something like this:

```
font-family: -apple-system, BlinkMacSystemFont,
             'helvetica neue', helvetica,
             ubuntu,
             roboto, noto,
             'segoe ui', arial,
             sans-serif;
```

or like this (for firefox-only products):

```
font: message-box;
```

Reply:
Is this preference true for general web stuff as well as in-product?

Reply:
Yes, that is correct, in-product we use system fonts. New Tab has been using system fonts since the AS beginnings and Firefox Accounts has also recently moved to system fonts, and they are planning to do the same for AMO.